vpr/src/route/route_timing.h

```diff
@@ -78,20 +109,34 @@ class ConnectionRouter {
         route_inf->backward_path_cost = cheapest->backward_path_cost;
     }
 
+    // Common logic from timing_driven_route_connection_from_route_tree and
+    // timing_driven_route_connection_from_route_tree_high_fanout for running
+    // connection router.
     t_heap* timing_driven_route_connection_from_route_tree(
         t_rt_node* rt_root,
         int sink_node,
         const t_conn_cost_params cost_params,
         t_bb bounding_box);
```

Comment:
It looks like the only difference between this and the public method is passing RouterStats. I feel like this should be named differently, as its confusing to overload these two functions.

Reply:
Renamed